stored_requests/backends/file_fetcher/fetcher.go

```diff
@@ -49,7 +50,12 @@ func (fetcher *eagerFetcher) FetchAccount(ctx context.Context, accountID string)
 			DataType: "Account",
 		}}
 	}
-	return accountJSON, nil
+
+	completeJSON, err := jsonpatch.MergePatch(accountDefaultsJSON, accountJSON)
```

Comment:
We should add test cases to cover this new change where default account information is provided to `FetchAccount` and it is properly merged and sent back.

Reply:
Added new test cases.